Client/Frontend/Browser/TabManager.swift

```diff
@@ -760,7 +760,7 @@ extension TabManager {
         }
         
         // Clean up any screenshots that are no longer associated with a tab.
-        imageStore?.clearExcluding(savedUUIDs)
+        _ = imageStore?.clearExcluding(savedUUIDs)
```

Comment:
Same, add `@discardableResult` to `clearExcluding`

Reply:
Done.